cellmlmanip/transpiler.py

```diff
@@ -58,6 +57,14 @@ def __init__(self, symbol_generator=None, number_generator=None):
         for tag_name in SIMPLE_MATHML_TO_SYMPY_NAMES:
             self.handlers[tag_name] = self._simple_operator_handler
 
+    def set_mathml_handler(mathml_operator, operator_class):
+        """Change how the transpiler handles a given mathml_operator.
+
+        :param mathml_operator: The name of a MathML operator e.g. 'exp', 'true' etc.
+        :param operator_class: A class that can handle the given operator e.g. sympy.exp, sympy.true etc.
+        """
+        SIMPLE_MATHML_TO_SYMPY_NAMES[mathml_operator] = operator_class
```

Comment:
Given the values are no longer names, this dict should be renamed e.g. to `SIMPLE_MATHML_TO_SYMPY_FUNCTIONS`

Reply:
Though they're not all functions either...

Reply:
technically they are classes actually